env.yaml

```diff
@@ -13,12 +13,7 @@ dependencies:
     - tqdm==4.50.2
```

Comment:
Shouldn't we delete this file then?

Reply:
I wouldn't since because Rahel and others might still use it for non-GNN related stuff